packages/adapter-next/siroc.config.ts

```diff
@@ -3,6 +3,7 @@ import { defineSirocConfig } from "siroc";
 export default defineSirocConfig({
 	rollup: {
 		output: {
+			exports: "named",
```

Comment:
Just out of curiosity / for my learning's sake, but why is this needed here? I guess it has to do with the named exports and export default in index 🤔

Reply:
Yes, exactly for that reason. Rollup printed the following message when a default export and named exports were mixed in the same file.

```
Entry module "src/index.ts" is using named and default exports together. Consumers of your bundle will have to use `chunk["default"]` to access the default export, which may not be what you want. Use `output.exports: "named"` to disable this warning
```

Since we read `plugin.default` when importing the plugin, I think it's okay that we export it as `default`.